### DISCUSSION TITLE
World Package 1.1.6버전이 업데이트 되었습니다

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#601] [1. Announcements]
안녕하세요.
World Package 1.1.6버전이 업데이트 되어서 안내드립니다.

### 안내
[패키지 업데이트 방법](https://studio.zepeto.me/kr/guides/updating-package)
[1.1.6 릴리즈 노트 보기](https://studio.zepeto.me/kr/guides/world-116)

 
 

### 신규 추가 기능

CharacterController - 캐릭터 제어 MotionV2 추가 
- 기존 제어 방식은 MotionV1으로 혼용 가능합니다.

![V2](https://user-images.githubusercontent.com/100118650/162382525-4e9f6fe0-4562-4840-a797-f6084cbdc8d8.png)

MotionV2 전용 모션은 다음과 같습니다.
- 더블 점프 (DoubleJump)
![DoubleJump](https://user-images.githubusercontent.com/100118650/162378161-fa819975-0e55-4baf-b433-1a9c2bf011a9.png)
- 착지 시 구르기 (LandingRoll)
![LandingRoll](https://user-images.githubusercontent.com/100118650/162378239-9f66d546-808a-4e48-b655-8928a76d1980.png)

- 달리기 중 자연스러운 방향 전환 (MoveTurn)
![MoveTurn](https://user-images.githubusercontent.com/100118650/162378243-7388f73c-6196-491f-b812-5b7fb703268f.png)

캐릭터 제어 MotionV2 관련 가이드 보기

[2. Character Control](https://studio.zepeto.me/guides/character-control)
[2-1. ZepetoCharacter](https://studio.zepeto.me/guides/zepetocharacter)
 
 

 
### 개선된 기능

- CharacterController - UIController_Horizontal, UIController_Vertical(고정형) 컨트롤러 프리팹 obsolete 처리
- CharacterController - ZepetoChracter 컴포넌트의 tryJump(Boolean)가 Jump Input을 받아오지 못하는 이슈 수정
- CharacterController - SetGesture(UnityEngine.AnimationClip) 반복 호출 시, 애니메이션이 다시 처음부터 재생되도록 수정
- CharacterController - Gesture State에 지정된 AnimationClip가 loop 아닐 경우, clip 재생 후 Gesture State 종료하도록 수정

제페토앱 3.11.3 이상부터 호환 되는 것 참고 부탁드립니다 😄

감사합니다.

**Comment by x4cf** [comment]
WOW!

**Comment by smilejsu82** [comment]
Good!
